proto/wasm/v1/scalars.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package wasm.v1;
+
+import "cosmos_proto/cosmos.proto";
+
+option (cosmos_proto.declare_scalar) = {
+    name: "RawContractMessage",
+    description: "A type representing a decimal number",
+    field_type: [SCALAR_TYPE_BYTES],
+    // Optionally we can add these 2 fields:
+
+    // Proposal 1:
+    gogo_customtype: "RawContractMessage", // This makes sure all fields that have `cosmos.scalar = "RawContractMessage"` also have `gogoproto.customtype = "RawContractMessage"`.
+    // TODO is this useful?
```

Comment:
Is this useful in your opinion @pyramation ?

Reply:
if we have `amino_encoding: "base64"` should be sufficient. 

I think if we standardize on putting all the info into a single field such as `amino_encoding`, then it helps minimize code branching to check other fields.